crates/uv/src/commands/project/mod.rs

```diff
@@ -74,8 +74,8 @@ pub(crate) enum ProjectError {
     #[error("The requested interpreter resolved to Python {0}, which is incompatible with the project's Python requirement: `{1}`")]
     RequestedPythonProjectIncompatibility(Version, RequiresPython),
 
-    #[error("The Python request from `{0}` resolved to Python {1}, which is incompatible with the project's Python requirement: `{2}`")]
-    DotPythonVersionProjectIncompatibility(String, Version, RequiresPython),
+    #[error("The Python version from `{0}` should be pinned to `{1}`")]
```

Comment:
I don't think we want to change the existing message, we should add a line _after_ it suggesting use of `uv python pin`.

Reply:
ACK.